src/js/components/plans/userInfo.tsx

```diff
@@ -72,13 +94,23 @@ const UserInfo = ({ user }: { user: User }) => {
         <p className="slds-p-top_small">
           <Trans i18nKey="credentialsHoldTime" count={token_minutes}>
             The credentials to your Salesforce org will only be held for{' '}
-            {{ token_minutes }} minutes or until your requested installation is
-            complete.
+            {{ count: token_minutes }} minutes or until your requested
```

Comment:
@davisagli I don't feel strongly about this change, and can revert it if you like. It matches the i18next pattern and docs more closely, which I like.

Reply:
@jgerigmeyer I see that you tried to save the translators work by updating all the existing translations, but I don't think it will actually help. When we submit a new round of translations to them, we just send them the current version of the English labels. My understanding is that they then load this into their system, which remembers the labels they've translated previously, and highlights the ones that need to be updated. So their starting point is the translation they produced last time, not the one that is in this repository if we've tweaked it.

Because of this I'd prefer not to make this change. (But I do like the idea of using `count` for new labels like this.)